tools/docker/Dockerfile

```diff
@@ -7,9 +7,10 @@ USER root
 
 # Tools
 RUN apt-get -qq update && \
-    apt-get -qq -y --no-install-recommends install ca-certificates > /dev/null && \
-    apt-get -qq update && \
-    apt-get -qq -y --no-install-recommends install gnupg software-properties-common > /dev/null
+    apt-get -qq -y --no-install-recommends install \
+      gnupg \
```

Comment:
While you're touching this, could you sort these packages alphabetically so they are consistent with the other large block of packages?

Reply:
Done. :)